Bottom: aabb7014ad38f47e53fe09832e18b9d42d833cf8
Top:    20cf969fc852aae4ea6dec5c4d5b50dce09f099b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-14 13:10:46 +0100

Refresh of add-debug-catch-to-triangle-ha

---

diff --git a/src/hid/common/hidgl.h b/src/hid/common/hidgl.h
index 4e86e27..1980195 100644
--- a/src/hid/common/hidgl.h
+++ b/src/hid/common/hidgl.h
@@ -46,6 +46,10 @@ hidgl_add_triangle (triangle_buffer *buffer,
                     GLfloat x2, GLfloat y2,
                     GLfloat x3, GLfloat y3)
 {
+  if (buffer->triangle_count == TRIANGLE_ARRAY_SIZE) {
+    printf ("OH CRAP\n");
+    *(char *)0 = 0;
+  }
   buffer->triangle_array [buffer->coord_comp_count++] = x1;
   buffer->triangle_array [buffer->coord_comp_count++] = y1;
   buffer->triangle_array [buffer->coord_comp_count++] = x2;
